LayoutTests/http/tests/site-isolation/iframe-dark-mode.html

```diff
@@ -0,0 +1,15 @@
+<!-- webkit-test-runner [ SiteIsolationEnabled=true ] -->
+<body bgcolor=blue>
+    <iframe onload="test()" src="http://localhost:8000/site-isolation/resources/dark-mode-background.html" frameborder=0></iframe>
+    <script>
+        if (window.testRunner)
+            window.testRunner.waitUntilDone();
+
+        function test() {
+            if (window.testRunner) {
+                window.testRunner.setUseDarkAppearanceForTesting(true);
```

Comment:
I'm curious why we need to add an internals API for this. Isn't it the same as setting `color-scheme: dark` on the root?

Reply:
I guess this is equivalent to setting m_useDarkAppearance rather than setting the document preference. Nevermind.

Reply:
It would also be good to have a test that uses CSS `color-scheme` or `<meta name="color-scheme">` to ensure that also propagates correctly.

Combinations of dark mode + `color-scheme: light` are also interesting.